test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -476,7 +476,7 @@ er
             @{ view = "Default"; widths = 4,7,5; variation = "narrow values"; values = [PSCustomObject]@{First=1;Second=2;Third=3}; wrap = $false; expectedTable = @"
 
 Long*Header2*Heade
-Long*********r3
+Long**********r3
```

Comment:
Please clarify why do old tests need to be updated?

Reply:
The old tests had the `center` aligned header `left` aligned, so they need to be shifted to be centered when it got wrapped.  I'll add a comment to the PR description.